cudamapper/src/utils.cpp

```diff
@@ -115,6 +121,193 @@ void print_paf(const std::vector<Overlap>& overlaps,
     }
 }
 
+#ifdef GW_BUILD_HTSLIB
+static inline void encode_cigar(bam1_t* alignment, const std::string& cigar)
```

Comment:
Pointers like `bam1_t* alignment` could still be constant pointers to non-constant values: `bam1_t* const alignment`

Reply:
ah true, done